File: libs/common/models/vw-location-publication.ts



export class VwLocationPublication {
    public locationPublicationId: number = 0;
    public locationPublicationStatus: string = '';
    public publishOnWebsiteFlag: boolean;
    public subRegion: string = '';
    public publicationDetailStatus: string = '';
    public locationId: number;
    public businessStartDate: Date;
    public businessEndDate: Date;

    constructor(init? : Partial<VwLocationPublication>) {
        
        Object.assign(this, init);
    }
}

File: apps/institution-api/Models/VwLocationPublication.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models
{
    public partial class VwLocationPublication
    {
        public int LocationPublicationId { get; set; }
        public string LocationPublicationStatus { get; set; }
        public bool? PublishOnWebsiteFlag { get; set; }
        public string SubRegion { get; set; }
        public string PublicationDetailStatus { get; set; }
        public int? LocationId { get; set; }
        public DateTime? BusinessStartDate { get; set; }
        public DateTime? BusinessEndDate { get; set; }
    }
}
